src/app/betting/feature/place-bet-page.component.ts

```diff
@@ -66,13 +77,111 @@ import { BetStatus } from '../../shared/util/supabase-helpers';
       />
     </label>
 
-    <!-- Bet terms -->
+    <!-- Bet Type Dropdown -->
+    <!-- eslint-disable-next-line -->
     <label class="flex flex-column gap-2 mt-5">
-      Bet Terms
-      <textarea rows="2" pInputTextarea [(ngModel)]="terms" [required]="true">
-      </textarea>
+      Bet Type
+      <p-dropdown
+        [options]="betTypes"
+        [(ngModel)]="selectedBetType"
+        optionLabel="betTypeString"
+        optionValue="betType"
+        styleClass="flex"
```

Comment:
Assuming the goal of `flex` is to make the dropdown stretch the full width of the page, I would change it to `w-full`. It has the same effect, and it makes the intent clearer.